### PR TITLE
translation fix for action

### PR DESCRIPTION
**Pull request by hasinaxp**
## Summary by CodeRabbit

- **New Features**
	- Enhanced translation process for action configurations.
	- Introduced serialization and deserialization for action data.

- **Bug Fixes**
	- Improved robustness in handling non-existent action types during translation.

- **Tests**
	- Added tests for translating action configurations, including edge cases for empty configurations and missing fields.

### CODE REVIEW

File: kairon/multilingual/processor.py [hunk 1]
```diff
@@ -149,8 +156,12 @@ def __save_bot_files(self, new_bot_id: str, nlu: TrainingData, domain: Domain, a
...
         """
         self.mp.delete_domain(bot=new_bot_id, user=self.user)
-
-        self.mp.save_integrated_actions(actions=actions, bot=new_bot_id, user=self.user)
+        ActionSerializer.deserialize(bot=new_bot_id,
+                                     user=self.user,
+                                     actions = actions,
+                                     other_collections_data=other_collections,
+                                     overwrite=True)
+        # self.mp.save_integrated_actions(actions=actions, bot=new_bot_id, user=self.user)
```
Comment (hiteshghuge) [inline kairon/multilingual/processor.py:164]: remove the comments

**Review by hiteshghuge — CHANGES_REQUESTED** [review]
